frontend/apps/marketing/src/app/[brand]/[locale]/layout.tsx

```diff
@@ -36,6 +37,15 @@ export default async function Layout({
   const localeConfig = SUPPORTED_LOCALES_MAP.get(locale);
   const theme = getMuiTheme(brand);
   const isDraftModeEnabled = (await draftMode()).isEnabled;
+  // Get Footer component based on brand
+  const getFooter = () => {
+    switch (brand) {
+      case Brand.CS_FOR_ALL:
+        return <FooterCSforAll locale={locale} />;
+      case Brand.CODE_DOT_ORG:
+        return <Footer locale={locale} />;
+    }
+  };
```

Comment:
Could we move this to `@/components/Footer`?

Reply:
I'm going to refactor the Code.org Footer in [this followup](https://codedotorg.atlassian.net/browse/CMS-964) to use the FooterMui component, could I move it then?

Reply:
Cool, added a note to my ticket!